src/Differ/MapDiffer.php

```diff
@@ -23,43 +25,17 @@
  */
 class MapDiffer implements Differ {
 
-	/**
-	 * @var bool
```

Comment:
But I don't when reading the code. I have to look for a line of code that initializes this class property, and then look the variable up that is used to initialize the property. Sure, all this is only a few lines below this property, but still. Please don#t remove such comments.